migrations/Migration20241219143807.ts

```diff
@@ -0,0 +1,33 @@
+import { Migration } from '@mikro-orm/migrations';
```

Comment:
I know it took some work, but I recommend to remove these SQLs. 
We can use automatic schema generation for entities: `npx mikro-orm schema:update --run` 

Having migration SQLs we have to maintain 2 things: migration SQLs + entities. It's easier to maintain only entities.